config/environments/production.rb

```diff
@@ -91,7 +91,11 @@
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
   if ENV['RAILS_LOG_TO_STDOUT'].present?
+<<<<<<< HEAD
```

Comment:
This is an issue from when you resolved the merge conflicts. You need to figure out what the code should be and remove the extra comment bits.

When you merged in the code, the lines between `<<<<< HEAD` and `======` were the lines on your current branch and the lines between `======` and `>>>>>> main` were the lines on the main branch you were pulling from

Reply:
I thought I had fixed them all but I must have missed one. I've fixed it now.

Reply:
you may be surprised how often that happens 😄